### DISCUSSION TITLE
Renaming levels and drawings in library

### DISCUSSION THREAD

**Discussion by cutydina** [opentoonz/opentoonz#6736] [Ideas]
### Is your feature request related to a problem? Please describe

Hello, any way that when a level is renamed it also renamed on the files? For example, here I change level name, but rute still same. Also, will be amazing to have a delete all unused drawings so we don't have folder full of unused drawings. Or maybe a note on the drawing that tell which drawings are been used? Hoping this will be possible, thanks in advance.

### Describe the solution you'd like

Change name but not rute:
![Captura de pantalla 2023-08-26 184822](https://github.com/opentoonz/opentoonz/assets/103970174/56ee7797-986e-4663-b731-da41104e138f)

My folder full of drawing i dont know what they are and if they are still been used?
![Captura de pantalla 2023-08-26 185002](https://github.com/opentoonz/opentoonz/assets/103970174/00c5c365-3976-4b69-a6a0-9184bdd40ddd)

**Comment by RodneyBaker** [comment]
Its a good habit to name levels when creating them as that will help identify them.
As for information such as when created or last modified... Right Click and choose Info (in various places in the program where we can right click to get such menus) and we'll get additional information about the file.

![image](https://github.com/opentoonz/opentoonz/assets/2576432/1ea82359-424a-4c2b-a9fd-5c490b817e9d)

Note that when we type in the title name for the level we aren't changing the file name... just the title.
To rename the file we want to use the Rename option in places such as the Browser Room.
There we can also show an icon... and the Info option is still available.

It'd be nice that if the info panel was open and we moved our selection to another file the info panel would automatically update.   I thought it did that but... apparently not.

![image](https://github.com/opentoonz/opentoonz/assets/2576432/eacb5f6a-e368-4de5-8ee2-84eef1b65afb)

Added:  A standard way to rename a file in Opentoonz is also to use Level > Save As.
Then to further propogate that name (if desired) we can turn on the option in Preferences to sync column names with level names.
